fintech-examples/fintech-server/src/main/resources/application.yml

```diff
@@ -19,10 +19,11 @@ mock:
 
 spring:
   datasource:
-    url: jdbc:h2:mem:test
-    username: SA
-    password:
-    driver-class-name: org.h2.Driver
+    url: jdbc:postgresql://localhost/open_banking?currentSchema=fintech
+    username: postgres
+    password: docker
+  liquibase:
+    change-log: classpath:master.xml
```

Comment:
Add `enabled: true`. No one on DEV will perform manual migrations. Liquibase should be disabled only on PROD (and even there it is questionable decision)